test/NinjaTools.FluentMockServer.API.Tests/Proxy/ComparasionVisitorTests.cs

```diff
@@ -0,0 +1,287 @@
+        {
+            // Arrange
+            var compare = new Headers(request);
+            var context = CreateContext(headers: contextHeaders);
+            var subject = new ComparasionVisitor(context);
+
+            // Act& Assert
+            var score = subject.Visit(compare);
+            subject.IsSuccess.Should().Be(isValid);
+            score.Should().Be(isValid ? 1 : 0);
+        }
+
+        [ItemNotNull]
+        public static IEnumerable<object[]> GetHeaderTestData()
+        {
+            yield return new object[]
+            {
+                new Dictionary<string, string[]>(),
+                new Dictionary<string, string[]>(),
+                true
+            };
+            yield return new object[]
+            {
+                null,
+                new Dictionary<string, string[]>(),
+                true
+            };
+            yield return new object[]
+            {
+                new Dictionary<string, string[]>
+                {
+                    {
+                        "Host", new[] {"Mock-Server"}
+                    }
+                },
+                new Dictionary<string, string[]>(),
+                false
+            };
+            yield return new object[]
+            {
+                new Dictionary<string, string[]>
+                {
+                    {
+                        "Host", new[] {"Mock-Server"}
+                    }
+                },
+                new Dictionary<string, string[]>
+                {
+                    {
+                        "Host", new[] {"Mock-Server"}
+                    }
+                },
+                true
+            };
+            yield return new object[]
+            {
+                new Dictionary<string, string[]>
+                {
+                    {
+                        "Some-Header", new[] {true.ToString(), 123.ToString()}
+                    }
+                },
+                new Dictionary<string, string[]>
+                {
+                    {
+                        "another.header", new[] {"with some other value"}
+                    }
+                },
+                false
+            };
+        }
+
+
+        private HttpContext CreateContext(string method = null, string path = null, Dictionary<string, string[]> headers = null, Stream requestBodyStream = null, QueryString? queryString = null)
+        {
+            var context = requestBodyStream is null
+                ? new DefaultHttpContext()
+                : new DefaultHttpContext
+                {
+                    Request =
+                    {
+                        Body = requestBodyStream
+                    }
+                };
+
+            var request = context.Request;
+            request.Method = method;
+            request.Path = new PathString(path);
+
+            if (queryString != null)
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: [Remove the unused private method 'CreateObject'.](https://app.codacy.com/manual/git_36/NinjaTools.FluentMockServer/pullRequest?prid=4802644)